app/components/map-interface/index.tsx

```diff
@@ -0,0 +1,119 @@
+import { makeStyles, useTheme } from "@rneui/themed"
+import MapView, { Region } from "react-native-maps"
+import MapStyles from "./map-styles.json"
+import React, { useRef } from "react"
+import { BusinessMapMarkersQuery } from "@app/graphql/generated"
+import { useI18nContext } from "@app/i18n/i18n-react"
+import MapMarker from "../map-marker"
+
+type Props = {
+  data?: BusinessMapMarkersQuery
+  userLocation?: Region
+  handleMapPress: () => void
+  handleMarkerPress: (_: MarkerData) => void
+  focusedMarker: MarkerData | null
+  handleCalloutPress: (_: MarkerData) => void
+}
+
+export type MarkerData = {
+  readonly __typename: "MapMarker"
+  readonly username?: string | null | undefined
+  readonly mapInfo: {
+    readonly __typename: "MapInfo"
+    readonly title: string
+    readonly coordinates: {
+      readonly __typename: "Coordinates"
+      readonly longitude: number
+      readonly latitude: number
+    }
+  }
+}
+
+export default function MapInterface({
```

Comment:
```suggestion
export default function MapComponent({
```

we generally use more the `Component` wording

Reply:
renamed